### PR TITLE
Add support for writing record with plain meta.

### PR DESCRIPTION
**Pull request by galxy25**
```
[octo@ValleyOfTheForge realms]$ cat plain.json 
{
	"tag": "tag"
}
[octo@ValleyOfTheForge realms]$ e3db write --PLAIN @plain.json test @test.json
cfd1eb11-0410-4aec-b09d-5339b37459f0
```

```
[octo@ValleyOfTheForge realms]$ e3db ls -d -j 
[
...
  {
    "meta": {
      "record_id": "cfd1eb11-0410-4aec-b09d-5339b37459f0",
      "writer_id": "2d742ace-3086-47d5-8bdd-f17327d05dba",
      "user_id": "2d742ace-3086-47d5-8bdd-f17327d05dba",
      "type": "test",
      "plain": {
        "tag": "tag"
      },
      "created": "2020-11-24T22:55:22.826854Z",
      "last_modified": "2020-11-24T22:55:22.826854Z",
      "version": "97ff29d0-c4b0-4b61-8b35-05be01eec083"
    },
    "data": {
      "test": "true"
    }
  }

]

```

### CODE REVIEW

**Review by efabens — COMMENTED** [review]
Is the `@` suymbol something we support elsewhere?

File: cmd/e3db/main.go [hunk 1]
```diff
@@ -217,10 +217,17 @@ func cmdWrite(cmd *cli.Cmd) {
 		HideValue: true,
 	})
 
+	plain := cmd.String(cli.StringOpt{
+		Name:      "PLAIN",
```
[the remark this answers is not in the document]
Comment (efabens) [reply cmd/e3db/main.go:221]: I'm also fine with --plain of -p but prefer meta

**Comment by efabens** [comment]
Now I see it. Wish i had known that previously kinda neat.

**Review by efabens — APPROVED** [review]

File: cmd/e3db/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (galxy25) [reply cmd/e3db/main.go]: Updated to meta

**Pull request merged**